modules/tinymce/src/plugins/legacyoutput/test/ts/browser/LegacyOutputPluginTest.ts

```diff
@@ -1,152 +1,177 @@
-      editor.execCommand('mceRemoveTextcolor', 'forecolor');
-      LegacyUnit.equal(editor.getContent(), '<p>text</p>');
-    });
-
-    suite.test('TestCase-TBA: LegacyOutput: Remove background color', (editor) => {
-      editor.setContent('<p><font style="background-color: red">text</font></p>');
-      LegacyUnit.setSelection(editor, 'font', 0, 'font', 4);
-      editor.execCommand('mceRemoveTextcolor', 'hilitecolor');
-      LegacyUnit.equal(editor.getContent(), '<p>text</p>');
-    });
-
-    suite.test('TestCase-TINY-4741: LegacyOutput: Convert bold to span if styling attributes are present on format removal', (editor) => {
-      editor.setContent('<p><b class="abc" style="color: red; font-size: 20px;" data-test="2">text</b></p>');
-      LegacyUnit.setSelection(editor, 'b', 0, 'b', 4);
-      editor.execCommand('bold');
-      LegacyUnit.equal(editor.getContent(), '<p><span class="abc" style="color: red; font-size: 20px;">text</span></p>');
-    });
-
-    suite.test('TestCase-TBA: LegacyOutput: Formats registered before loading initial content', () => {
-      const formats = formatsCell.get();
-      LegacyUnit.equal(formats.bold[0], { inline: 'b', remove: 'all', deep: true, split: true, preserve_attributes: [ 'class', 'style' ] });
-      LegacyUnit.equal(formats.italic[0], { inline: 'i', remove: 'all', deep: true, split: true, preserve_attributes: [ 'class', 'style' ] });
-      LegacyUnit.equal(formats.underline[0], { inline: 'u', remove: 'all', deep: true, split: true, preserve_attributes: [ 'class', 'style' ] });
-      LegacyUnit.equal(formats.fontname[0], { inline: 'font', toggle: false, attributes: { face: '%value' }, deep: true, split: true });
-    });
-
-    TinyLoader.setupLight((editor, onSuccess, onFailure) => {
-      Pipeline.async({}, Log.steps('TBA', 'LegacyOutput: Test legacy formatting', suite.toSteps(editor)), onSuccess, onFailure);
-    }, {
-      plugins: 'legacyoutput',
-      indent: false,
-      base_url: '/project/tinymce/js/tinymce',
-      font_formats: 'Arial=arial,helvetica,sans-serif;',
-      setup: (editor) => {
-        // Store the formats on `PostRender`, which is fired before the initial editor content is loaded
-        editor.on('PostRender', () => {
-          formatsCell.set({ ...editor.formatter.get() });
-        });
-      }
-    }, success, failure);
-  }
-);
+describe('browser.tinymce.plugins.legacyoutput.LegacyOutputPluginTest', () => {
+  const formatsCell = Cell<any>({});
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'legacyoutput',
+    indent: false,
+    base_url: '/project/tinymce/js/tinymce',
+    font_formats: 'Arial=arial,helvetica,sans-serif;',
+    setup: (editor) => {
+      // Store the formats on `PostRender`, which is fired before the initial editor content is loaded
+      editor.on('PostRender', () => {
+        formatsCell.set({ ...editor.formatter.get() });
+      });
+    }
+  }, [ Plugin, Theme ]);
```

Comment:
```suggestion
  }, [ Plugin, Theme ], true);
```

This makes the editor focus once it's setup, which is effectively the same as what the `editor.focus()` call was trying to do below.

Reply:
done